Makefile

```diff
@@ -1,27 +1,59 @@
-.PHONY: all patches app clean
+.PHONY: all clean
 
-all: patches app
+OUT_FOLDER      := out
+
+CIA_OUT_FOLDER  := $(OUT_FOLDER)/cia_out
+3DSX_OUT_FOLDER := $(OUT_FOLDER)/3dsx_out
+
+3DS_OUT	        := 3ds
+CIA_OUT         := cias
+
+LUMA_OUT	    := luma/titles
 
 FRIEND_TITLE_ID	:= 0004013000003202
 ACT_TITLE_ID	:= 0004013000003802
+SSL_TITLE_ID    := 0004013000002F02
+JUXT_ID_JPN     := 000400300000BC02
+JUXT_ID_USA     := 000400300000BD02
+JUXT_ID_EUR     := 000400300000BE02
```

Comment:
That one was on me, I was pretty indecisive about what I should refer to Miiverse as in all this, should I rename the "juxt" patches folder too or keep that one the same?

Reply:
The Juxtaposition branding and name should only really be used in contexts that aren't referring to official products made by Nintendo. That is why we have repositories named like `miiverse-api`, and `juxtaposition-ui`. `miiverse-api` implements the Miiverse API created by Nintendo/Hatena, it is not our own creation. `juxtaposition-ui` uses our branding and name since it _is_ our own creation

If you are patching the Miiverse title, then it's (title IDs, executable names, etc) referring to an official product made by Nintendo

Reply:
These have been changed